lib/src/exception/exception.ts

```diff
@@ -2,28 +2,67 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+import {codeFrameColumns, SourceLocation} from '@babel/code-frame';
+
 import {SourceSpan} from './span';
 
 /**
  * An exception thrown by Sass.
  */
 export class SassException extends Error {
   /**
-   * @param message - The error message.
+   * @param [message] - The error message.
    * @param [span] - The source span associated with the error.
-   * @param [stack] - The stack trace associated with the error.
+   * @param [trace] - The trace associated with the error.
    */
-  constructor(message?: string, readonly span?: SourceSpan, stack?: string) {
+  constructor(message?: string, readonly span?: SourceSpan, trace?: string) {
```

Comment:
One alternative to building the highlight yourself would be having the compiler provide a formatted message along with a `CompileFailure`/`LogEvent`. What do you think?

Reply:
I like the idea of having a formatted message provided by the compiler.

Reply:
If that's the case, we'd also need to tell the compiler whether to colorize the formatted message

Reply:
I updated the protocol here: https://github.com/sass/embedded-protocol/pull/49. If that looks good I can also update dart-sass-embedded.